dm/worker/server.go

```diff
@@ -402,15 +405,15 @@ func (s *Server) QueryWorkerConfig(ctx context.Context, req *pb.QueryWorkerConfi
 	if err != nil {
 		resp.Result = false
 		resp.Msg = errors.ErrorStack(err)
-		log.Errorf("[worker] query worker config error %v", errors.ErrorStack(err))
+		log.L().Error("fail to query worker config", zap.String("request", "query worker config"), zap.Stringer("payload", req), zap.Error(err))
 		return resp, nil
 	}
 
 	rawConfig, err := workerCfg.From.Toml()
 	if err != nil {
 		resp.Result = false
 		resp.Msg = errors.ErrorStack(err)
-		log.Errorf("[worker] marshal worker config error %v", errors.ErrorStack(err))
+		log.L().Error("fail to marshal worker config", zap.String("request", "query worker config"), zap.Stringer("wotker from config", &workerCfg.From), zap.Error(err))
```

Comment:
```suggestion
		log.L().Error("fail to marshal worker config", zap.String("request", "query worker config"), zap.Stringer("worker from config", &workerCfg.From), zap.Error(err))
```
`zap.Stringer("worker from config", &workerCfg.From)` database password will be logged here ?

Reply:
you can check it, Stringer would call json marshal